aws/efs/cmd/efs-provisioner/volumemetadata.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"os"
+	"path"
+	"strconv"
+
+	"encoding/json"
+	"github.com/golang/glog"
+	"io/ioutil"
+)
+
+const (
+	metadataFile = ".kube-efs-provisioner-metadata"
+)
+
+type volumeMetadata struct {
+	GID              string `json:"gid"`
```

Comment:
Is there a reason why this can't just be a `uint32`? Or, if it can be optional a `*uint32`? The string conversions seem unnecessary.

Reply:
Definitely don't use 0, sorry wasn't thinking about it being an actual group ID :grimacing:. Could you use `*uint32` in this case?

Reply:
I would likely not have used a string here. But it's fine, so, I'm just going to resolve it.